extensions/amp-mustache/0.2/amp-mustache.js

```diff
@@ -132,7 +132,7 @@ export class AmpMustache extends AMP.BaseTemplate {
    */
   purifyAndSetHtml_(html) {
     const diffing = isExperimentOn(self, 'amp-list-diffing');
-    const body = purifyHtml(html, diffing);
+    const body = purifyHtml(html, this.win.document, diffing);
```

Comment:
@choumx should we use a PWA safe method `this.getAmpDoc()`

Reply:
This one is OK since we're just checking for the AMP spec on the `<html>` tag. We can consider a way to do this via AmpDoc instead of window.document though.